control_panel_api/management/commands/grant_aws_roles_permission_to_read_inline_policies.py

```diff
@@ -0,0 +1,81 @@
+import json
+import logging
+
+import boto3
+from botocore.exceptions import ClientError
+from django.conf import settings
+
+from control_panel_api.management.commands.dry_runnable import DryRunnable
+from control_panel_api.models import User
+from control_panel_api.services import READ_INLINE_POLICIES_POLICY_NAME as POLICY_NAME
+
+
+IAM = boto3.client('iam')
+POLICY_ARN = f"{settings.IAM_ARN_BASE}:policy/{POLICY_NAME}"
+POLICY_DOCUMENT = {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "CanReadUserRolesInlinePolicies",
+            "Effect": "Allow",
+            "Action": [
+                "iam:GetRolePolicy"
+            ],
+            "Resource": [
+                f"{settings.IAM_ARN_BASE}:role/{settings.ENV}_user_*",
+            ],
+        },
+    ],
+}
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_policy():
+    try:
+        IAM.create_policy(
+            PolicyName=POLICY_NAME,
+            PolicyDocument=json.dumps(POLICY_DOCUMENT),
+        )
+        logger.info(f'IAM policy "{POLICY_NAME}" created')
```

Comment:
Consider removing this log line, it will make output a bit verbose. The usual guideline is to log only if something is unexpected. So since you're already logging "Creating policy", there's no real need to log "created" (or even "already exists"), just unexpected things like errors.

Reply:
Removed 👍